docs/pythreejs-example.rst

```diff
@@ -0,0 +1,89 @@
+=================
+PyThreeJs Example
+=================
+
+Thebe can display output from pythreejs_. The examples are taken from the
+pythreejs documentation and are licensed BSD 3 Clause.
+
+.. _pythreejs: https://github.com/jupyter-widgets/pythreejs
+
+Press the "Activate" button below to connect to a Jupyter server:
+
+.. raw:: html
+
+   <button id="activateButton" style="width: 120px; height: 40px; font-size: 1.5em;">
+     Activate
+   </button>
+   <script>
+   var bootstrapThebe = function() {
+       thebelab.bootstrap();
+   }
+   document.querySelector("#activateButton").addEventListener('click', bootstrapThebe)
+   </script>
+
+.. raw:: html
+
+   <script type="text/x-thebe-config">
+     {
+       requestKernel: true,
+       binderOptions: {
+         repo: "jupyter-widgets/pythreejs",
+         ref: "2.2.1",
```

Comment:
I figure we should pin these to specific versions of the repo2docker file somehow.